sites/docs/src/content/docs/guidelines/components/modules.md

```diff
@@ -750,6 +750,37 @@ Input data SHOULD be referenced with the `modules_testdata_base_path` parameter:
 file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true)
 ```
 
+### Configuration of ext.args in tests
+
+Module nf-tests SHOULD use a single `nextflow.config` to supply `ext.args` to a module, by taking the settings from the
+params block defined in the `when` block of a test.
```

Comment:
To me the sentence was difficult to read that is why I tried to change it a little
```suggestion
Module nf-tests SHOULD use a single `nextflow.config` to supply `ext.args` to a module. They can be defined in the `when` block of a test under the `params` scope.
```